CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to this project will be documented in this file.
 - CloudSQ: fixed Terraform change detection when backup is disabled
 - Allow multiple CIDR blocks in the ip_range for Apigee Instance
 - Add prefix to project factory SA bindings 
+- Add support for Private Service Connect and Reginal Managed Proxy subnets for `net-vpc` module
```

Comment:
Typo: Reginal -> Regional Managed Proxy. Maybe explain that regional managed proxy subnets are for the new L7 Regional External Load Balancer?

Reply:
done

Reply:
One last typo: "Private Service Connect and **Reginal** Managed" -> "Private Service Connect and **Regional** Managed"

Reply:
It is fixed